SETUP.md

```diff
@@ -16,7 +16,7 @@ You can do Code.org development using OSX, Ubuntu, or Windows (running Ubuntu in
      yarn --version  # --> 1.6.0
      ```
 1. If using HTTPS: `git clone https://github.com/code-dot-org/code-dot-org.git`, if using SSH: `git@github.com:code-dot-org/code-dot-org.git`
-1. `gem install bundler`
+1. `gem install bundler 1.17`
```

Comment:
```suggestion
1. `gem install bundler -v 1.17`
```
Wasn't sure whether the `-v` flag is required; it was present in the first few examples I looked up, maybe we should include it.